### PR TITLE
Fixed: Passing of CLI args, Added: CLI args, Removed: Comments

### PR DESCRIPTION
**Pull request by monfron**
- I didnt like the passing of cli args...the parameter list was getting too large
- new cli args for setting the update interval and measurement stop

Signed-off-by: monfron <simon.fronhoefer@gmail.com>

### CODE REVIEW

File: mapago-client.go [hunk 1]
```diff
@@ -15,32 +16,34 @@ import "github.com/protocollabs/mapago/control-plane/ctrl/shared"
...
 	msmtType := flag.String("msmt-type", "tcp-throughput", "tcp-throughput or udp-throughput")
-	msmtStreams := flag.Int("msmt-streams", MSMT_STREAMS, "setting number of streams")
-	msmtLAddr := flag.String("msmt-listen-addr", "127.0.0.1", "localhost or userdefined addr")
-	msmtCSize := flag.Int("msmt-call-size", DEF_BUFFER_SIZE, "msmt application buffer in bytes")
-	
+	msmtStreams = flag.Int("msmt-streams", MSMT_STREAMS, "setting number of streams")
+	msmtListenAddr = flag.String("msmt-listen-addr", "127.0.0.1", "localhost or userdefined addr")
+	msmtCallSize = flag.Int("msmt-call-size", DEF_BUFFER_SIZE, "msmt application buffer in bytes")
+	msmtUpdateTime = flag.Uint("msmt-update-interval", 2, "msmt update interval in seconds")
+	msmtStopTime = flag.Uint("msmt-stop-interval", 10, "complete msmt time in seconds")
```
Comment (hgn) [inline mapago-client.go]: ohne msmt

```
--streams <n>
--listen-addr <addr> # das ist doch nicht die listen address?! -> --addr
--call-size ??? Nope! Wie bei iperf
--update-interval --> --ctrl-update-interval <n>
--stop-interval ? Kein Interval -> --measurement-time
```

File: mapago-client.go [hunk 2]
```diff
@@ -525,3 +500,37 @@ func validateDiscovery(req *shared.DataObj, rep *shared.DataObj) error {
...
+		if index != len(msmtInfoRep.Data.DataElements) - 1 {
+			evaluationStr.WriteString(", ")			
+		}
+	}
+
+	// 6. add suffix
+	evaluationStr.WriteString(" ]")
+
+	fmt.Println(evaluationStr.String())
+}
```
Comment (hgn) [inline mapago-client.go:535]: super! Richtig gut!

**Review by hgn — CHANGES_REQUESTED** [review]
schau dir mal die paar anmerkungen an, sonst super! Danke

File: mapago-client.go (hunk 1 above)
Comment (monfron) [reply mapago-client.go]: ok fixed. listen-addr stand aber iwo noch im std

**Comment by monfron** [comment]
New pull Request will be opened....

**Pull request closed without merging**